perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -157,7 +157,6 @@ def _pywb_static_dir_location():
 
```

Comment:
(Actually, from your deployment note, I surmise you think it's better to keep it in there, to make future changes easier?)